app/code/Magento/CompareListGraphQl/etc/schema.graphqls

```diff
@@ -0,0 +1,42 @@
+# Copyright © Magento, Inc. All rights reserved.
+# See COPYING.txt for license details.
+
+type ComparableItem {
+    productId: ID! @doc(description: "Product Id")
+    name: String! @doc(description: "Product name")
+    sku: String! @doc(description: "Product SKU")
+    priceRange: PriceRange! @doc(description: "Product prices")
+    canonical_url: String @doc(description: "Product URL")
+    images: [ProductImage]! @doc(description: "Product Images")
+#    values: [ProductAttribute]! @doc(description: "Product comparable attributes")
```

Comment:
@AndreyChorniy Please resolve the comments from above.

Reply:
As this PR is in progress it was commented temporary, because _ProductAttribute_ type was not described in **schema.graphqls** And this is not clear how it should work. 
It will be resolved after discussion with architecture team.

Reply:
@nrkapoor, Can you look along with @AndreyChorniy, @Usik2203 on this?

Reply:
for consistency purpose, you may use https://github.com/magento/architecture/blob/master/design-documents/graph-ql/custom-attributes-container.md